### PR TITLE
Depricated code removal for old models

### PR DESCRIPTION
**Pull request by nagarikantipavan**

### CODE REVIEW

File: src/test/java/net/authorize/util/LuhnTest.java [hunk 1]
```diff
@@ -1,72 +1,56 @@
 package net.authorize.util;
```
Comment (gnongsie) [inline src/test/java/net/authorize/util/LuhnTest.java]: This file can be deleted.

File: src/main/java/net/authorize/util/HttpClient.java [hunk 2]
```diff
@@ -85,44 +87,48 @@ private static HttpPost createHttpPost(Environment env, Transaction transaction)
...
+		httpPost.getParams().setBooleanParameter(CoreProtocolPNames.USE_EXPECT_CONTINUE, false);
+
+		// set the TCP connection timeout
+		httpPost.getParams().setIntParameter(HttpConnectionParams.CONNECTION_TIMEOUT, httpConnectionTimeout);
+		// set the time out on read-data request
+		httpPost.getParams().setIntParameter(HttpConnectionParams.SO_TIMEOUT, httpReadTimeout);
+
+		httpPost.setHeader("Content-Type", "text/xml; charset=utf-8");
+		httpPost.setEntity(new StringEntity(transaction.toXMLString(), HTTP.UTF_8));
+		/* } */
```
Comment (gnongsie) [inline src/main/java/net/authorize/util/HttpClient.java]: Remove lines 90 - 131

File: src/main/java/net/authorize/util/HttpClient.java [hunk 3]
```diff
@@ -135,24 +141,23 @@ private static HttpPost createHttpPost(Environment env, Transaction transaction)
...
+	 * // aim/sim if(transaction instanceof net.authorize.aim.Transaction ||
+	 * transaction instanceof net.authorize.sim.Transaction) {
+	 * 
+	 * String decodedResponseData = URLDecoder.decode(responseString, HTTP.UTF_8);
+	 * 
+	 * 
+	 * responseMap = ResponseParser.parseResponseString(decodedResponseData); }
+	 * 
+	 * return responseMap; }
+	 */
```
Comment (gnongsie) [inline src/main/java/net/authorize/util/HttpClient.java]: Remove commented code.

File: src/main/java/net/authorize/util/HttpClient.java [hunk 4]
```diff
@@ -161,49 +166,48 @@ private static HttpPost createHttpPost(Environment env, Transaction transaction)
...
+	 * String cleanResponseString =
+	 * XmlUtility.descapeStringForXml(rawResponseString);
+	 * 
+	 * responseMap = HttpClient.createResponseMap(transaction, cleanResponseString);
+	 * } catch (Exception e) { LogHelper.warn(logger,
+	 * "Exception getting response: '%s': '%s', '%s'", e.getMessage(), e.getCause(),
+	 * Arrays.toString(e.getStackTrace())); } }
+	 * 
+	 * return responseMap; }
+	 */
```
Comment (gnongsie) [inline src/main/java/net/authorize/util/HttpClient.java]: Remove commented code.

File: src/main/java/net/authorize/util/HttpClient.java [hunk 5]
```diff
@@ -213,181 +217,167 @@ private static HttpPost createHttpPost(Environment env, Transaction transaction)
...
+	 * 
+	 * int mark = rawResponseString.indexOf("<?xml"); if(mark == -1){ return null; }
+	 * 
+	 * response.parseString(rawResponseString.substring(mark,rawResponseString.
+	 * length())); if(response.IsAccessible() == false){ return null; } } catch
+	 * (Exception e) { LogHelper.warn(logger,
+	 * "Exception getting response: '%s': '%s', '%s'", e.getMessage(), e.getCause(),
+	 * Arrays.toString(e.getStackTrace())); } }
+	 * 
+	 * return response; }
```
Comment (gnongsie) [inline src/main/java/net/authorize/util/HttpClient.java]: Remove commented code.

File: src/test/java/net/authorize/api/controller/test/ApiCoreTestBase.java [hunk 6]
```diff
@@ -185,71 +171,73 @@ public static void tearDownAfterClass() throws Exception {
...
-		//initialize counter
+		mockContext = new Mockery();
+
+		// initialize counter
 		counter = random.nextInt((int) Math.pow(2, 24));
 		counterStr = getRandomString("");
 
 		now = Calendar.getInstance().getTime();
-		nowString = DateUtil.getFormattedDate(now, ReportingDetails.DATE_FORMAT);
+		// nowString = DateUtil.getFormattedDate(now, ReportingDetails.DATE_FORMAT);
```
Comment (gnongsie) [inline src/test/java/net/authorize/api/controller/test/ApiCoreTestBase.java]: Remove this line.

File: src/test/java/net/authorize/util/LuhnTest.java (hunk 1 above)
Comment (nagarikantipavan) [reply src/test/java/net/authorize/util/LuhnTest.java]: file has been deleted

File: src/main/java/net/authorize/util/HttpClient.java (hunk 2 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/util/HttpClient.java]: code has been removed

File: src/main/java/net/authorize/util/HttpClient.java (hunk 3 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/util/HttpClient.java]: code has been removed

File: src/main/java/net/authorize/util/HttpClient.java (hunk 4 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/util/HttpClient.java]: code has been removed

File: src/main/java/net/authorize/util/HttpClient.java (hunk 5 above)
Comment (nagarikantipavan) [reply src/main/java/net/authorize/util/HttpClient.java]: code has been removed

File: src/test/java/net/authorize/api/controller/test/ApiCoreTestBase.java (hunk 6 above)
Comment (nagarikantipavan) [reply src/test/java/net/authorize/api/controller/test/ApiCoreTestBase.java]: code has been removed

**Comment by gnongsie** [comment]
Ready to Merge.

**Pull request merged**